app/dashboard/templates/ens/ens.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'ens/base.html' %}
+{% load static %}
+{% block 'scripts' %}
+
+<script src="https://cdn.jsdelivr.net/npm/ethereumjs-util@5.1.5/dist/index.min.js"></script>
+<script src="{% static 'v2/js/shared.js' %}"></script>
+<script src="{% static 'ens/index.js' %}"></script>
+      {% block 'ens_script' %}{% endblock %}
+
+{% endblock %}
+<!-- Main -->
+{% block 'main' %}
+<section>
+            {% include 'shared/current_balance.html' %}
```

Comment:
Fix indentation throughout to use 2-space indentation and modify all existing indents to conform to the indentation based on their depth.